scenarios/kubernetes_e2e.py

```diff
@@ -500,6 +571,45 @@ def create_parser():
         action='append',
         default=[],
         help='Send unrecognized args directly to kubetest')
+
+    # aws
+    parser.add_argument(
+        '--aws', action='store_true', help='E2E job runs in aws')
+    parser.add_argument(
+        '--aws-profile',
+        default=(
+            os.environ.get('AWS_PROFILE') or
+            os.environ.get('AWS_DEFAULT_PROFILE') or
+            'default'
+        ),
+        help='Profile within --aws-cred to use')
+    parser.add_argument(
+        '--aws-role-arn',
+        default=os.environ.get('KOPS_E2E_ROLE_ARN'),
```

Comment:
`kops-aws-updown` is running in a separate account: `--aws-role-arn=arn:aws:iam::660757973107:role/job-kops-aws-updown`

It was my intention to shift them all to this style at some point (the equivalent of GCP projects), but I didn't finish that conversion.